app/server/controllers/login.ts

```diff
@@ -14,8 +13,24 @@ import * as superAgent from 'superagent';
 const logger = Logger.getLogger('login.js');
 const idamUrlString: string = config.get('idam.url');
 const idamClientId: string = config.get('idam.client.id');
+const enableDummyLogin = config.get('enableDummyLogin') === 'true';
 
-function getLogin(getRedirectUrl: (protocol: string, hostname: string) => string, idamPath: string) {
+function redirectToLogin(req: Request, res: Response) {
+    return res.redirect(Paths.login);
+}
+
+function getLogout(req: Request, res: Response) {
+  const sessionId: string = req.session.id;
+  req.session.destroy(error => {
+    if (error) {
+      logger.error(`Error destroying session ${sessionId}`);
+    }
+    logger.info(`Session destroyed ${sessionId}`);
+    return res.redirect(Paths.login);
```

Comment:
Probably one for later, but we should probably be destroying the session in IDAM too, but not sure what that involves.